Makefile

```diff
@@ -146,12 +146,21 @@ TEST_APK_PROJECTS = \
 	tests/CodeGen-Binding/Xamarin.Android.JcwGen-Tests/Xamarin.Android.JcwGen-Tests.csproj \
 	tests/locales/Xamarin.Android.Locale-Tests/Xamarin.Android.Locale-Tests.csproj
 
+TEST_APK_PROJECTS_RELEASE = \
+	src/Mono.Android/Test/Mono.Android-Tests.csproj
+
 # Syntax: $(call BUILD_TEST_APK,path/to/project.csproj)
```

Comment:
This comment should be updated, as you're now providing two values to the `define`.